.github/SECURITY.md

```diff
@@ -0,0 +1,122 @@
+# Oppia Vulnerability Disclosure Process
+
+This vulnerability disclosure process describes how we accept and respond to security vulnerabilities from both Oppia developers and others. Our process follows 4 steps: Report, Investigate, Remediate, and Disclose.
+
+## Report
+
+Reporters should email [security@oppia.org](mailto:security@oppia.org) with:
+
+* A description of the problem.
+* Steps we can follow to reproduce the problem.
+* Affected versions.
+* If known, mitigations for the problem.
+
+We will respond within 3 days of the reporter’s email to acknowledge receipt of their report. Here is a template acknowledgement email:
+
+```
+Hi $REPORTER,
+
+Thank you for reporting this problem to us. We are currently investigating and will reach out sometime in the next 7 days once we have decided how to move forward or if we have any questions.
+
+Thanks,
+
+$OPPIA_SECURITY_TEAM_MEMBER
+```
+
+## Investigate
+
+Immediately upon receiving a report of a security problem, a member of Oppia’s security team (the tech leads for now) will assemble a vulnerability response team (VRT). This team should:
+
+* Include an Oppia tech lead.
+* Include developers (most likely 1-2) with expertise in the part of the app affected by the problem.
+* Include as few developers as possible to avoid premature publication of the problem.
+
+The tech lead will designate one VRT member as the VRT lead responsible for driving our response. The VRT lead will immediately (ideally within 24 hours) investigate the report and classify it as:
+
+* Won’t fix: The app is working as intended, the bug is not important enough for us to spend resources fixing, or the requested feature is not something we plan to implement.
+* **Bug**: The problem identified is legitimate, but it is not a security issue. It will be resolved through our normal bug-fixing process.
+* **Feature request**: The report identifies a new feature that should be implemented. It will be handled through our normal feature-development process.
```

Comment:
I assume this is also not a security-related feature?

Reply:
This could be a security feature. For example, if someone reported as a "vulnerability" the fact that we don't offer users a way to restrict logins based on IP address, that could be filed as a feature request (assuming we wanted to add that functionality)

Reply:
Okay, I see.